app/views/jobseekers/sessions/new.html.slim

```diff
@@ -2,7 +2,7 @@
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    h1.govuk-heading-xl = t(".title")
+    h1.govuk-heading-l = t(".title")
```

Comment:
Sad

Reply:
But I trust your judgement

Reply:
just for consitency with the two sign in pages

Reply:
![Screenshot 2021-02-25 at 13 16 40](https://user-images.githubusercontent.com/1792451/109158845-cda0b700-776b-11eb-92ef-f28bcb2a318f.png)
![Screenshot 2021-02-25 at 13 16 30](https://user-images.githubusercontent.com/1792451/109158851-ced1e400-776b-11eb-8c3a-42c2b50ff268.png)